RELEASE.md

```diff
@@ -0,0 +1,12 @@
+# Release Process
+
+The release process is mostly automated and requires running only a few commands. After commiting, pushing, tagging and releasing the changes, a GitHub Action will publish all npm packages.
```

Comment:
Is it a good idea to explicitly say, that this procedure works on any branch (and not only on `main`) or is that trivial?

Reply:
Not sure, would that not confuse future maintainers?

Reply:
I don't know, let's skip it then